### ISSUE TITLE
How to reproduce the baselines such as CDA and Dropout?

### ISSUE THREAD

**Issue by richhh520** [McGill-NLP/bias-bench#18]
Hi dear authors,

In the stereoset_debias.py, for example, when I set args.model to be CDABertForMaskedLM or DropoutBertForMaskedLM, the produced results are the same. It seems that actually the debiased models are not loaded. Could you help me with this? Thanks a lot!
It seems that in the batch_jobs folder, there are some scripts to reproduce, could you please provide a README? I will be really appreciated!

Richy

**Comment by richhh520** [comment]
btw, how to reproduce **self-debias** results? Thankss vey much!!

**Comment by 123qwe-ux** [comment]
Hello, dear author,
I have the same question. I don't know how to evaluate debiasing techniques like CDA. Following the instructions provided in your README file, it seems that the evaluation results haven't changed. Could you please tell me how to resolve this? Thank you very much, and I wish all your papers get accepted.

**Comment by ncmeade** [comment]
Hi @richhh520 and @123qwe-ux ,

Sorry for the very late response!

W.r.t the CDA and Dropout models, unfortunately, I don't have the original checkpoints so I can't share them here. However, you should be able to re-train/fine-tune a model using `experiments/run_mlm.py`/`experiments/run_clm.py`. If your code is using a newer version of Transformers than what we use in the package, you should be able to extract the augmentation functions into newer scripts (see [L573](https://github.com/McGill-NLP/bias-bench/blob/main/experiments/run_mlm.py#L574C76-L574C76)). You can reference `batch_jobs/counterfactual_augmentation.sh` and `batch_jobs/dropout.sh` to see the parameters we used for training.

W.r.t Self-Debias, what experiments are you having difficulties running? You should be able to use any of experiment scripts for debiasing for running Self-Debias models (for instance, `experiments/stereoset_debias.py`).

**Comment by ncmeade** [comment]
Closing this now due to inactivity. Feel free to reopen for further discussion.
